modules/ossm-update-app-sidecar.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-update-app-sidecar_{context}"]
+= Updating your application pods
+
+For {ProductName} 1.0.1, run the following commands to update the sidecars on your application pods to the most recent version. 
```

Comment:
I think this should probably say something like,

```md
If updating from a previous version of {ProductName}, you will need to update the sidecar containers on your application pods to the new version.  You can run the following command to trigger a rolling update for deployments configured to use automatic injection:

$ oc patch deployment/<deployment> -p '{"spec":{"template":{"metadata":{"annotations":{"kubectl.kubernetes.io/restartedAt": "'$(date -Iseconds)'"}}}}}'

This simply updates an annotation on the deployment's pod template, which will trigger a new rollout, with updated sidecar containers.
```

Reply:
My guess is this only triggers a rolling update with the rolling update strategy (default). There's also the option to use the recreate strategy. https://docs.openshift.com/container-platform/3.3/dev_guide/deployments/deployment_strategies.html#rolling-strategy

Reply:
good point.  maybe we should just state that this will trigger a redeployment of the existing pods.

Reply:
Ok, I've made the following updates. 

```diff
-For {ProductName} 1.0.1, run the following commands to update the sidecars on your application pods to the most recent version. 
+If you're updating {ProductName} from a previous version, you must update the sidecars on your application pods to the most recent version. 
```

```diff
-Then run following command to trigger a rolling restart of the deployment
+Then, run following command to redeploy the pods:
```

Does that work for you @rcernich and @brian-avery ?

Reply:
Sounds good to me. Thanks